src/sentry/static/sentry/app/components/events/sdkUpdates.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+
+import {t} from 'app/locale';
+
+class EventSdkUpdateSuggestion extends React.Component {
+  static propTypes = {
+    suggestion: PropTypes.object.isRequired,
```

Comment:
Can we use `PropTypes.shape` to define the object a bit more?

Reply:
done